portfolio/src/main/webapp/script.js

```diff
@@ -19,5 +19,24 @@ window.onload = () => {
   }).then((greeting) => {
     console.log('Adding greeting to dom: ' + greeting);
     document.getElementById('greeting-container').innerText = greeting;
-  })
-}
\ No newline at end of file
+  });
+  
+  fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages);
```

Comment:
Ideally you should remove this from the final version, but if this is still in progress, it's fine to leave as-is.